pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -854,6 +855,29 @@ protected BookieAffinityGroupData internalGetBookieAffinityGroup() {
         }
     }
 
+    public void setNamespaceBundleAffinity (String bundleRange, String destinationBroker) {
+        if (destinationBroker != null) {
+            if (!this.isLeaderBroker()) {
+                LeaderBroker leaderBroker = pulsar().getLeaderElectionService().getCurrentLeader().get();
+                String leaderBrokerUrl = leaderBroker.getServiceUrl();
+                try {
+                    URL redirectUrl = new URL(leaderBrokerUrl);
+                    URI redirect = UriBuilder.fromUri(uri.getRequestUri()).host(redirectUrl.getHost())
+                            .port(redirectUrl.getPort()).replaceQueryParam("authoritative",
+                                    false).build();
+
+                    // Redirect
+                    log.debug("Redirecting the rest call to leader - {}, bundleRange - {}", redirect, bundleRange);
```

Comment:
always try to keep `log.debug` under `if` -> `if(log.isDebugEnabled)`

Reply:
done